src/Modules.js

```diff
@@ -17,11 +23,17 @@ module.exports = {
   'blend': require('./modules/Blend'),
   'import-image': require('./modules/ImportImage'),
   'overlay': require('./modules/Overlay'),
+  'edge-detect': require('./modules/EdgeDetect'),
+  'fisheye-gl': require('./modules/FisheyeGl'),
+  'gamma-correction': require('./modules/GammaCorrection'),
   'gradient': require('./modules/Gradient'),
+  'import-image': require('./modules/ImportImage'),
   'invert': require('image-sequencer-invert'),
   'ndvi-colormap': require('./modules/NdviColormap'),
   'colorbar': require('./modules/Colorbar'),
-  'histogram': require('./modules/Histogram'),
   'gamma-correction': require('./modules/GammaCorrection'),
   'convolution': require('./modules/Convolution'),
-}
```

Comment:
Missing new line at the end of file.Probably if you are using sublime then you need to change your text editor settings so just google it.